Sources/MapboxNavigation/NavigationView.swift

```diff
@@ -66,22 +66,24 @@ open class NavigationView: UIView {
         navigationMapView.delegate = delegate
     }
     
-    var tileStoreLocation: TileStoreConfiguration.Location? = .default
-    private var _navigationMapView: NavigationMapView? = nil
-    
     // :nodoc:
-    public lazy var navigationMapView: NavigationMapView = {
-        _navigationMapView?.frame = bounds
-        let navigationMapView = _navigationMapView ?? NavigationMapView(frame: bounds, tileStoreLocation: tileStoreLocation)
-        navigationMapView.isHidden = false
-        navigationMapView.translatesAutoresizingMaskIntoConstraints = false
-        
-        navigationMapView.delegate = delegate
-        navigationMapView.navigationCamera.viewportDataSource = NavigationViewportDataSource(navigationMapView.mapView,
-                                                                                             viewportDataSourceType: .active)
-        
-        return navigationMapView
-    }()
+    public var navigationMapView: NavigationMapView {
```

Comment:
Ability to set `navigationMapView` directly allows to prevent blinking while performing transition between preview and active navigation modes.